jdaviz/core/unit_conversion_utils.py

```diff
@@ -0,0 +1,398 @@
+from astropy import units as u
+import itertools
```

Comment:
`itertools` is in stdlib

```suggestion
import itertools

from astropy import units as u
```